pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/requests.tsx

```diff
@@ -16,19 +16,19 @@ export default function (props: GraphDashboardProps) {
     </LineGraph>,
 
     <LineGraph title="Slow Raft Proposals" sources={storeSources}>
-      <Axis units={AxisUnits.Count} label="proposals">
+      <Axis units={AxisUnits.Count} label="requests">
```

Comment:
@nvanbenschoten do you think these are good labels?

Reply:
(y-axis label on time series graph)

Reply:
I'm not sure I get why they're being changed, to be honest. They were more accurate before.

Reply:
ok, swapped out the names in requests to the more detailed versions, and also fixed the linting error. will squash and change the commit message before merging. also waiting for andrew to take a look.